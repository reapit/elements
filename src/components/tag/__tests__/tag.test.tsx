import { render } from '@testing-library/react'
import { Tag, TagGroup } from '..'

describe('Tag', () => {
  it('should render tag group properly and match snapshot', () => {
    const { asFragment } = render(
      <TagGroup>
        <Tag>Tag</Tag>
      </TagGroup>,
    )
    expect(asFragment()).toMatchSnapshot()
  })
  it('should render standalone tag properly and match snapshot', () => {
    const { asFragment } = render(<Tag as="span">Tag</Tag>)
    expect(asFragment()).toMatchSnapshot()
  })
})
