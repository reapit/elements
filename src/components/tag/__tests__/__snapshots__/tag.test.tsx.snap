// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Tag > should render standalone tag properly and match snapshot 1`] = `
<DocumentFragment>
  <span
    class="mocked-styled-0 el-standalone-tag"
  >
    Tag
  </span>
</DocumentFragment>
`;

exports[`Tag > should render tag group properly and match snapshot 1`] = `
<DocumentFragment>
  <ul
    class="mocked-styled-2 el-tag-group"
  >
    <li
      class="mocked-styled-1 el-tag"
    >
      Tag
    </li>
  </ul>
</DocumentFragment>
`;
